index.html

```diff
@@ -0,0 +1,174 @@
+          <a href="#">Logout</a>
+          <a href="#">My Page</a>
+          <a href="#">English</a>
+        </div>
+      </section>
+      <nav class="navbar navbar-expand-md navbar-light">
+        <div class="container-fluid">
+          <a class="navbar-brand d-none d-md-inline" href="index.html">
+            <img class="nav-img" src="images/logo1.jpg" alt="">
+          </a>
+          <button
+            class="navbar-toggler border-0 fs-1 fw-bold"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarSupportedContent"
+            aria-controls="navbarSupportedContent"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <i class="fa-solid fa-bars"></i>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+              <li class="ms-auto d-md-none pe-3">
+                <i class="fa-solid fa-x"  data-bs-toggle="collapse" data-bs-target=".navbar-collapse.show"></i>
+              </li>
+              <li class="nav-item p-2">
+                <a class="nav-a active-nav" href="index.html">Home</a>
+              </li>
+              <li class="nav-item p-2">
+                <a class="nav-a" href="about.html">About</a>
+              </li>
+              <li class="nav-item p-2">
+                <a class="nav-a join-us" href="#">Join Us</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+      <section id="main-header">
+        <h3 class="main-header-h3">"Hello! Sharing world"</h3>
+        <h1 class="main-header-h1">ProCoder Full-Stack Diploma 2021</h1>
+        <p class="main-header-p1">
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+          Necessitatibus quos ullam labore at placeat eligendi dolore asperiores
+          in maiores!
+        </p>
+        <h2 class="main-header-h2">2021.12.15 To 2022.6.15</h2>
+        <p class="main-header-p2">
+          @ National Museum of Korea, Art Center Nabi and more
+        </p>
+      </section>
+    </header>
+
+    <main>
+      <section id="main-program">
+        <h2 class="program-h2">Main Program</h2>
+        <ul class="list-unstyled main-program-ul">
+          <li class="program-card">
+            <i class="fas fa-chalkboard-teacher fa-2x icon"></i>
+            <h4 class="program-card-h4">Lecture</h4>
+            <p class="program-card-desc">
+              Learn from the artists from various countries about the art of
+              drawing.
+            </p>
+          </li>
+          <li class="program-card">
+            <i class="fas fa-tv fa-2x icon"></i>
+            <h4 class="program-card-h4">Exhibition</h4>
+            <p class="program-card-desc">
+              Learn from the artists from various countries about the art of
+              drawing.
+            </p>
+          </li>
```

Comment:
- [ ] Each of the program cards should be wrapped in an `article` tag to contain the elements.

Reply:
Done